<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameConfig</class>
 <widget class="QDialog" name="GameConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DDCLauncher - Game Upgrader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="EngineLayout">
     <item>
      <widget class="QLabel" name="lbExecutable">
       <property name="text">
        <string>Executable:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ExecutableLayout">
       <item>
        <widget class="QLineEdit" name="inputExecutable"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpenExec">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDefaultExec">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="Sections">
     <item>
      <layout class="QVBoxLayout" name="NickSection">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbNick">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Nick:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbNickEnable">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Enable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbNickColorful">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Colorful</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="inputNick">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Buttons">
     <item>
      <spacer name="ButtonsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
